src/core/Akka.Remote/RemoteDeploymentWatcher.cs

```diff
@@ -21,30 +16,31 @@ namespace Akka.Remote
     /// Responsible for cleaning up child references of remote deployed actors when remote node
     /// goes down (crash, network failure), i.e. triggered by Akka.Actor.Terminated.AddressTerminated
     /// </summary>
-    internal class RemoteDeploymentWatcher : ActorBase, IRequiresMessageQueue<IUnboundedMessageQueueSemantics>
+    internal class RemoteDeploymentWatcher : ReceiveActor, IRequiresMessageQueue<IUnboundedMessageQueueSemantics>
```

Comment:
@alexvaluyskiy @Horusiath that was my reasoning - make the code easier to understand.